documents/motions/06.2021.1 - Regional Organizations.md

```diff
@@ -0,0 +1,58 @@
+# SUBMISSION OF PROPOSED MOTION
+
+**Motion number:** 6.2021.1  
+**Subject:** Regional Organizations  
+**Intent:** Procedure of acknowledgement of Regional Organizations and description of their rights and duties  
+**Submitted by:** WCA Board  
+**Date:** February 28, 2021  
+
+# Motion
+
+
+The WCA shall acknowledge Regional Organizations as Members to support the Objectives of the WCA in specific regions.
+
+1. Rights of Regional Organizations:
+   1. Regional Organizations may represent the WCA in the region that they are acknowledged for, with activities that support the Objectives of the WCA.
+   2. Regional Organizations may recognize a separate organization to act on its behalf in a subsection of the region of which it covers. This organization shall have the full rights and responsibilities of the Regional Organization.
```

Comment:
Shouldn't they also have the Duties of the RO too? If they have the full rights, doe that mean sub-ROs can also recongnise sub-ROs?

Also, does this mean all "sub-regional organisations" should also be submitting an annual report - wouldn't it be simpler to incorporate this into the report of their parent RO?

Reply:
We could change it to "Regional Organizations *recognized directly by the WCA* may recognize a separate organization to act on its behalf in a subsection of the region of which it covers. This organization shall have the full rights and responsibilities of the Regional Organization." to stop such recursion.

I think the Duties also makes sense, @thewca/board ?

Reply:
We believe the new wording should clarify your concerns